collab-document/src/document_data.rs

```diff
@@ -27,7 +27,8 @@ pub fn default_document_data() -> DocumentData {
   let text_type = PARAGRAPH_BLOCK_TYPE.to_string();
 
   let mut blocks: HashMap<String, Block> = HashMap::new();
-  let mut meta: HashMap<String, Vec<String>> = HashMap::new();
+  let mut children_map: HashMap<String, Vec<String>> = HashMap::new();
+  let mut text_map: HashMap<String, String> = HashMap::new();
 
   // page block
   let page_id = nanoid!(10);
```

Comment:
would be better to craete a function that generate the nanoid id? You can mark the function as #[inline].

Reply:
why nanoid!(10)?